src/js/components/CommandPanel/CommandPanel.tsx

```diff
@@ -228,16 +242,27 @@ const CommandPanel = ({setLogs}: {setLogs: TSetLogs}) => {
             return;
         }
 
-        const hexWithoutComments = removeComments(hex);
-
+        let preparedHex;
```

Comment:
rename origin state

[hex, setHex] to [dump, setDump]

here rename to `hex`

Reply:
done